float32[4] selfiestate
